android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/LanguageResource.java

```diff
@@ -14,6 +22,17 @@
   protected String version;
   protected String helpLink;
 
+  // JSON keys
+  public static String LR_PACKAGE_ID_KEY = "packageID";
+  public static String LR_RESOURCE_ID_KEY = "resourceID";
+  public static String LR_RESOURCE_NAME_KEY = "resourceName";
+  public static String LR_LANGUAGE_ID_KEY = "languageID";
+  public static String LR_LANGUAGE_NAME_KEY = "languageName";
+  public static String LR_VERSION_KEY = "version";
+  public static String LR_HELP_LINK_KEY = "helpLink";
```

Comment:
If possible, these should be private as hopefully we can keep the implementation internal to the class.